Cht-android coverage report (#640)
medic/cht-docs: content/en/core/guides/android/development-setup.md

### Testing

> To execute unit tests, run: `make test` (static checks are also executed).

Reviewer: I think it makes even more sense without the parens, eg:

```suggestion
To execute unit tests and static analysis, run: `make test`.
```
